test/test_fake_quant.py

```diff
@@ -80,10 +92,14 @@ def test_numerical_consistency(self, device, Q):
         np.testing.assert_allclose(Y, Y_prime.cpu(), rtol=tolerance, atol=tolerance)
```

Comment:
I think one day we should unify which assert we want to call as well, right now we are using both self.assertEqual and np.testing.assert